ambari-web/test/models/stack_service_test.js

```diff
@@ -112,11 +113,11 @@ describe('App.StackService', function () {
         isInstallable: true,
         result: false
       },
-      {
-        serviceName: 'MAPREDUCE2',
-        isInstallable: true,
-        result: true
-      },
+      // {
+      //   serviceName: 'MAPREDUCE2',
+      //   isInstallable: true,
+      //   result: true
+      // },
```

Comment:
Possibility of uncommenting later ?
if not, can we remove ?

Reply:
App.StackService.coSelected code needs to be fixed (we are not clear about YARN and MR would need to exist together or not). Once done this will be uncommented